### PR TITLE
Update Get-CsOnlineUser.md - Minor changes 2

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -589,7 +590,7 @@ The following attributes are no longer relevant to Teams and have been dropped f
 1. LineURI set via OnPrem AD.
 2. Direct Routing numbers assigned to users via Set-CsUser.
 
-In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via OnPrem AD. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.
+In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via OnPrem AD and OnPremLineURIManuallySet attribute has been dropped. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.
```
Comment (tseward) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:593]: I would rephrase this to something along the lines of 'set via OnPrem AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used.'.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 1 above)
Comment (dariomws) [reply skype/skype-ps/skype/Get-CsOnlineUser.md:593]: Agree

Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:593]:
```suggestion
In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.
```

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 1 above)
Comment (JohanFreelancer9) [reply skype/skype-ps/skype/Get-CsOnlineUser.md:593]: For consistency, please also change "Numbers" to "numbers" in the last sentence (two instances).

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -589,7 +590,7 @@ The following attributes are no longer relevant to Teams and have been dropped f
 1. LineURI set via OnPrem AD.
 2. Direct Routing numbers assigned to users via Set-CsUser.
 
-In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via OnPrem AD. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.
+In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:593]:
```suggestion
In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing numbers can be distinguished from Calling Plan numbers by looking at the FeatureTypes attribute.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -147,6 +147,9 @@ Accept wildcard characters: False
 ```
 
 ### -Filter
+
+**Note: This immediate section only applies to TPM 2.6 and lower**
```
Comment (tiburd) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:151]:
```suggestion
**Note: This immediate section only applies to TPM 2.6 and earlier**
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -155,9 +158,9 @@ For example, a filter that returns only users who have been enabled for Enterpri
 
 `{EnterpriseVoiceEnabled -eq $True}`
 
-**Note:**
 
-The following updates are applicable for TeamsOnly customers using Microsoft Teams PowerShell version 3.0.0 and above.
+
+The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and above.
```
Comment (tiburd) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:163]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and later.
```

Comment (tiburd) [inline skype/skype-ps/skype/Get-CsOnlineUser.md]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and later.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 5]
```diff
@@ -270,6 +273,9 @@ Accept wildcard characters: False
 ```
 
 ### -Identity
+
+**Note: This immediate section only applies to TPM 2.6 and lower**
```
Comment (tiburd) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:277]:
```suggestion
**Note: This immediate section only applies to TPM 2.6 and earlier**
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 6]
```diff
@@ -449,7 +455,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## OUTPUTS
 
 ### Note:
-The following updates are applicable for TeamsOnly customers using Microsoft Teams PowerShell version 3.0.0 and above.
+The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and above.
```
Comment (tiburd) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:458]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and later.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 7]
```diff
@@ -589,7 +596,7 @@ The following attributes are no longer relevant to Teams and have been dropped f
 1. LineURI set via OnPrem AD.
 2. Direct Routing numbers assigned to users via Set-CsUser.
 
-In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via OnPrem AD. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.
+In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing numbers can be distinguished from Calling Plan numbers by looking at the FeatureTypes attribute.
```
Comment (tiburd) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:599]:
```suggestion
In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises Active Directory. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing numbers can be distinguished from Calling Plan numbers by looking at the FeatureTypes attribute.
```
